### ISSUE TITLE
Weights get loaded with too many significant figures

### ISSUE THREAD

**Issue by typicalTYLER** [typicalTYLER/dumbbellppl#2]
and will sometimes fail validation if it goes into scientific notation
![image](https://user-images.githubusercontent.com/6762150/34137933-d25407cc-e432-11e7-9131-4cb3c0cdbf33.png)

started happening after the move to sqlalchemy

**Comment by typicalTYLER** [comment]
Possibly related warning

```
SAWarning: Dialect sqlite+pysqlite does *not* support Decimal objects natively, and SQLAlchemy must convert from floating point - rounding errors and other issues may occur. Please consider storing Decimal numbers as strings or integers on this platform for lossless storage.
  'storage.' % (dialect.name, dialect.driver))
```

**Comment by typicalTYLER** [comment]
Should be fixed as of commit 342b1a7c06ceb927c1bd338a782cb390375c74fa
